ChangeLog.md

```diff
@@ -1,6 +1,33 @@
 
 # Change Log
 
+## Version 10.25.0
```

Comment:
Will the bump to the version constant in AzCopy be in a future PR? We could just address that in this PR.

Reply:
Whoops! Let me get that done here.